local-setup/github-action-config.json

```diff
@@ -16,7 +16,9 @@
         "-w",
         "2001",
         "-r",
-        "3443"
+        "3443",
+        "-h",
```

Comment:
-h is often associated with "help" therefore I suggest taking a different letter. For example -a